website/content/docs/get-started/developer-qs.mdx

```diff
@@ -1,19 +1,19 @@
 ---
 layout: docs
-page_title: Your First Secret
+page_title: Developer Quickstart
 description: Learn how to store and retrieve your first secret.
 ---
 
-# Create Your First Secret
+# Developer Quickstart
 
-In this quick start, we’ll explore how to use Vault client libraries inside our application code to store and retrieve your first secret value. Vault takes the security burden away from developers by providing a secure, centralized secret store for an application’s sensitive data: credentials, certificates, encryption keys, and more.
+In this quickstart, we’ll explore how to use Vault client libraries inside your application code to store and retrieve your first secret value. Vault takes the security burden away from developers by providing a secure, centralized secret store for an application’s sensitive data: credentials, certificates, encryption keys, and more.
```

Comment:
FYI I learned during this task that quickstart can be spelled "Quickstart" or "Quick Start", both are correct. While I'm more of a "quickstart" person myself, I changed it to be "quick start" here to match how we use it on the Learn guide nav. So if you care about that consistency you may want to change it back. But either way is fine.

Reply:
yes, personally I'm a fan of 'quickstart' but that's a good point about making it consistent with Learn.

Reply:
Looks like you've changed it back everywhere except the nav bar, which is arguably where the consistency matters most. 😉

Reply:
Fixed.

Reply:
FIxed